recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -31,13 +33,24 @@ def get_built_distribution_names_and_subdirs(recipe_dir, variant):
                 'clobber_sections_file': clobber_file
             }
             break
+    
+    # if feedstock don't have new recipe.yaml
+    # use default conda_build.api.render for meta.yaml
+    if not rattler_has_recipe(recipe_dir):
```

Comment:
You should use `conda-forge.yml`'s value for configuring build-tool here.

Reply:
Initially I thought about extracting the value from `conda-forge.yaml` but the `retry_upload_or_check` don't pass any feedstocks paths so I have only recipe_dir.
I can try and go up one level from recipe_dir and assume that it will be the feedstock itself, but I'm not sure if it's the right way of doing it.

Another idea is to have additional option for `upload_or_check_non_existence` ( something like --rattler-build ) that will be passed from conda-smithy.

wdyt?

Reply:
I think we should pass in a feedstock_dir to these functions.